.changeset/yellow-debt-v2-confidence-calibration.md

```
@@ -0,0 +1,42 @@
+---
+"yellow-debt": minor
+---
+
+# Scanner output schema v2.0 — failure_scenario field, audit-synthesizer
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)